Powershell-PoC-Environment-Setup.ps1

```diff
@@ -20,12 +20,81 @@
     or lawsuits, including attorneys’ fees, that arise or result from the use or distribution of the Sample Code.
     Please note: None of the conditions outlined in the disclaimer above will supersede the terms and conditions contained
     within the Premier Customer Services Description.
+.NOTES
+    AUTHOR(S): Paulo Marques
+    CONTRIBUTOR(S): Preston K. Parsard
+    KEYWORDS: PoC, Deployment, NEW 0.00.00.0008
+    
+    LICENSE:
+
+    The MIT License (MIT)
+    Copyright (c) 2016 Paulo Marques
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software. 
+
+.LINK
+    https://mit-license.org/license.txt
+
+.LINK
+    https://www.powershellgallery.com/packages/WriteToLogs
+#>
+
+<#
+***************************************************************************************************************************************************************************
+REVISION/CHANGE RECORD	
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------

```

Comment:
Simplify the change log, this is too long and too complex to manage, looks great but practically speaking it is a lot of work to maintain, make it something like this:

Change Log:

	* Included New-RamdomString function
	* Added code to prompt user for subscription name
	* Added Start-transcript to have full log of the execution
	...
	etc

We will add the detailed changes into de the README.MD file, so please keep the details there. The README.MD will look like this:

Change Log

xx/xx/xx
	Initial version

xx/xx/xx
	Change 1
	Change 2